src/tests/GraphTest.hpp

```diff
@@ -64,54 +69,49 @@ class GraphTest : public Test {
             
             /* set interval to inf so this won't be killed by timeout */
             gtc->interval = numeric_limits<double>::max();
+            auto stats = g->grab_stats();
+            std::apply(print_stats, stats);
+            insertionProb = 3300;
```

Comment:
Make it more generic, like parameters or environment variables (passed during run time via `-d`)?

Reply:
I'm unfamiliar with how to obtain this information. Is there an example somewhere?

Reply:
Yes. The former one is simple: just add four more parameters to the constructor of GraphTest. The latter is a little trickier: when you run `bin/main ... -dInsertProb=3300`, gtc will generate a record and you can get it by 
```
    if(gtc->checkEnv("InsertProb")){
       insertionProb = atoi((gtc->getEnv("InsertProb")).c_str());
```